cluster-api-gcp/CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contributing Guidelines
+
+## Build
+
+```bash
+$ cd $GOPATH/src/k8s.io/
+$ git clone git@github.com:kubernetes/kube-deploy.git
+$ cd kube-deploy/cluster-api-gcp/
+$ go build
+```
+
+This will create a binary `cluster-api-gcp` that you can use to manage a GCP cluster.
+
+## Developing
+
+When making changes to the machine controller, it's generally a good idea to delete any existing cluster created with an older version of the cluster-api.
+
+```bash
+$ ./cluster-api-gcp delete
+```
+
+After making changes to the machine controller or the actuator, you need to follow these two steps:
+
+1. Rebuild the machine-controller image.
```

Comment:
You also need to change the version to something custom in pods.go and the Makefile. Otherwise you might be overriding the existing official image or not using your latest image.

Reply:
That's a good gotcha. Fixed.

Reply:
I may have missed it but I do not see the instructions to update the versions.